internal/page/donor/how_would_certificate_provider_prefer_to_carry_out_their_role.go

```diff
@@ -58,9 +58,14 @@ type howWouldCertificateProviderPreferToCarryOutTheirRoleForm struct {
 func readHowWouldCertificateProviderPreferToCarryOutTheirRole(r *http.Request) *howWouldCertificateProviderPreferToCarryOutTheirRoleForm {
 	carryOutBy, err := actor.ParseCertificateProviderCarryOutBy(page.PostFormString(r, "carry-out-by"))
 
+	email := page.PostFormString(r, "email")
+	if carryOutBy.IsPaper() {
+		email = ""
+	}
```

Comment:
I ended up adding this in as we [validate](https://github.com/ministryofjustice/opg-data-lpa-store/blob/main/lambda/create/validate.go#L25) on this field not being present in the lpa store for paper CPs.